src/Microsoft.AspNetCore.Razor.Evolution/TagHelpers/TagHelperFeature.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Evolution.Legacy;
+
+namespace Microsoft.AspNetCore.Razor.Evolution.TagHelpers
+{
+    internal class TagHelperFeature : IRazorEngineFeature
```

Comment:
This needs to be public but we need to discuss/finalize our `ITagHelperDescriptorResolver` contracts to do that (A legacy piece that's internal).

Also, horribad name but couldn't think of anything better from the original prototype.